### ISSUE TITLE
How to use @DocumentExample with List<String> as return type?

### ISSUE THREAD

**Issue by iorlas1** [stoicflame/enunciate#358]
I have a property with the following get method:

```
    @DocumentationExample(value="role1", value2 = "role2")
    public List<String> getRoles(){
        return roles;
    }
```
The generated json-documentation I get from this states:
  "roles" : [ "role1", "role1" ],

It seems that the List<String> return type is not handled the way that I expected. I thought I would get
  "roles" : [ "role1", "role2" ], or possibly  "roles" : [ "role1", ... ], 

Magnus

**Comment by stoicflame** [comment]
Thanks for the report. Assigned to 2.4.

**Comment by stoicflame** [comment]
Fixed at 96c5d5f.

**Comment by stoicflame** [comment]
[Enunciate 2.4 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.4.0).
